.github/workflows/update-feedback-labels.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Update contributor feedback labels on comment'
+on: 'issue_comment'
+
+jobs:
+  feedback:
+    if: github.event.issue && contains(github.event.issue.labels.*.name, 'needs feedback')
```

Comment:
Would this run for both open and closed issues?

Reply:
Just open ones.

Reply:
@tammullen I just tested that and it does process on closed ones. Nice catch.

Reply:
I wonder if that is maybe ok that it does. Right now we actually don't have closed issues with has feedback so it might be helpful for catching when comments come in on closed issues, although maybe the stalebot itself would trigger it on closed issues? Actually that makes me wonder how this action would work on the open issues the stalebot would be commenting on with the initial 7 day warning? Would the stalebot trigger `issue_comment`?

Reply:
> Right now we actually don't have closed issues with has feedback

I'm wondering whether we would want to have the bot re-open a closed issue on comment if it had either of the feedback labels and apply `needs feedback`.

> Would the stalebot trigger issue_comment?

Most likely yes. I'll look at adding a check for this.

I think what we should do is address the stalebot comment issue in this PR then discuss the comment on closed issue handling on a separate issue/PR.

Reply:
@tammullen I added logic to exclude comments by `github-actions` and limit to open issues.

Reply:
> I think what we should do is address the stalebot comment issue in this PR then discuss the comment on closed issue handling on a separate issue/PR.

Sounds good!